Add files via upload

alisherr003 opened the pull request.

alisherr003 committed:
Add files via upload

Marip7 reviewed (changes requested):
Hello, Alisher, 

There are some issues that need to be fixed according to the task.

For the entire work:
Schema name missing before table name.
The code standards were missed (tabs).

For specific tasks:
1.1
The WHERE clause must include a condition to filter by category 'animation'.

1.2
The task was skipped; it needs to be added.

1.3
According to the task, you need to add a condition that the movie was released after 2015.

1.4
You hardcoded the IDs. You should filter by the categories 'drama', 'travel', 'documentary' instead.

2.1
Please add the staff names.

2.2
OK

3.1
Your query returns NULLs in the columns last_movie_year and gap_since_last_movie, which makes the result incorrect.

3.2 
The max_gap column is calculated incorrectly, as it calculates the maximum across the entire table.

For example, in the first row of your result:
148 EMILY DEE 30

select fa.film_id, f.release_year
from public.film_actor fa 
inner join public.film f on f.film_id = fa.film_id
where fa.actor_id = 148
order by f.release_year
The max_gap is 9 years.

alisherr003 committed:
Add files via upload

Good evening Marina
Here is attached the corrected version of SQL HW.
If there is any mistake, please let me to know.
